jenkins/Dockerfile.ubuntu16

```diff
@@ -35,7 +35,7 @@ RUN add-apt-repository ppa:deadsnakes/ppa && \
     openjdk-8-jdk python3.6 python3-pip tzdata git
 
 RUN ln -s /usr/bin/python3.6 /usr/bin/python
-RUN python -m pip install pytest sre_yield requests
+RUN python -m pip install pytest sre_yield requests pandas numpy pyarrow
```

Comment:
nit: `numpy` should be installed as a pandas dependency, that means we do not need to add it here

Reply:
Could pip hanle the dependence installation ?

Reply:
yep, at least it is capable for this simple scenario

Reply:
updated